jsapp/js/components/library/myLibraryRoute.es6

```diff
@@ -78,18 +84,34 @@ class MyLibraryRoute extends React.Component {
       );
     }
 
+    // Modifying Dropzone like this feels a bit weird, should we add a wrapper?
```

Comment:
It sucks, but a wrapper would probably be a bit better solution, as if we update `react-dropzone` npm package then there is a tiny chance this will break.